src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(1000)];
+        for (Figure figure: figures) {
+            figure = new FigureSupplier().get();
```

Comment:
new FigureSupplier is created on each loop iteration - let's fix this

Reply:
add